lib/jekyll/commands/doctor.rb

```diff
@@ -52,17 +51,16 @@ def conflicting_urls(site)
           urls = collect_urls(urls, site.pages, site.dest)
           urls = collect_urls(urls, site.posts.docs, site.dest)
           urls.each do |url, paths|
-            if paths.size > 1
-              conflicting_urls = true
-              Jekyll.logger.warn "Conflict:", "The URL '#{url}' is the destination" +
-                " for the following pages: #{paths.join(", ")}"
-            end
+            next unless paths.size > 1
+            conflicting_urls = true
+            Jekyll.logger.warn "Conflict:", "The URL '#{url}' is the destination" \
+              " for the following pages: #{paths.join(", ")}"
           end
           conflicting_urls
         end
 
-        def fsnotify_buggy?(site)
-          return true if !Utils::Platforms.osx?
+        def fsnotify_buggy?(_site)
```

Comment:
Didn't want to do anything that would appear to change the API, since this is a public method.

I suppose it could be changed to `def fsnotify_buggy?(_ = nil)` and then not pass `site`?